styleguide/source/_patterns/01-molecules/09-search-results/search-results-best-match.twig

```diff
@@ -0,0 +1,19 @@
+{#
+ # @file search-results-best-match.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<div class="layout layout_one_up">
+  {% include 'atoms-h6' with { 'class' : 'search-result_label', 'content' : 'Best Match' } %}
+  <div class="search-result">
+    <div class="search-result_left">
+      <div class="search-result_left-inner">
+        {% include 'atoms-h4' with { 'class' : 'search-result_title', 'content' : 'Lorem Ipsum Dolor Amet Consectetur Adipiscing Elit of Laoreet Pharetra' } %}
+        {% include 'atoms-paragraph' with { 'class' : 'search-result_description', 'content' : 'Sed massa augue, sollic itudiny vel metus a, posuere erat. In vulputate feugiat neque, sed aliquam juh tasga hugit.' } %}
+        {% include 'atoms-link' with { 'class' : 'link-blue search-result_link', 'content' : 'Learn More' } %}
```

Comment:
instead of a link, this should just be a 'fake' link with a span or a p tag i think